start/sw/web-app/rails.md

```diff
@@ -6,33 +6,24 @@ order: 6
 
 # Installing Ruby and Rails with rbenv
 
-The first step is to install dependencies for Ruby.
+The first step is to install dependencies for Ruby and rbenv.
 
 ```
-sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
+sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison libcurl-devel sqlite-devel rbenv rbenv-build-rbenv
```

Comment:
I think there are a few dependencies missing, the updated instructions here https://developer.fedoraproject.org/tech/languages/ruby/ruby-installation.html list more things. I am quite sure that without, at least, the Perl dependencies, Using some Ruby Versions will fail without them.

Reply:
Hmm, it's been a while since I ran it on my desktop and I recall having some perl issues but then I just ran it in a container and it worked without.

Though now that I look at the bigger picture, why don't we simply instruct users for Rails to follow https://developer.fedoraproject.org/tech/languages/ruby/ruby-installation.html first? Or use the system installed Ruby by default?

Reply:
The perl issues are almost Ruby version specific. (yes, Ruby, not rbenv...)

Instructing users to follow that one is also a possibility, I wanted to suggest de-duplicating this way, but then I read you actually use the system rbenv, which we don't describe on that page. It can be of course brought over and just linked here for users to first follow those instructions.

And I would recommend against using system Ruby for Rails. In my entirely personal opinion, working with Rails without rbenv or other way to keep some semblance of a stable environment for more than a year gets very annoying.

Reply:
Let's leave this be for now and think about it at some later PR for rbenvs.